### PR TITLE
File november.md committed.

### PR DESCRIPTION
**Pull request by Translationmanagement**

### CODE REVIEW

File: _lang/ja/_developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ Android アプリの \[**プッシュ通知設定**] で \[**Firebase**] を選
 次に、Android Studio プロジェクトを開き、Firebase 送信者 ID を使用して、`braze.xml` または `BrazeConfig` 内で FCM トークンの自動登録を有効にします。
 
 {% tabs local %}
-{% tab braze.xml %}
+braze.xml %}
```
Comment (josh-mccrowell-braze) [inline _lang/ja/_developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration.md:124]:
```suggestion
{% tab braze.xml %}
```

Comment (iaraltkorn) [reply _lang/ja/_developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration.md]: @josh-mccrowell-braze looks good, thanks! we can merge!

**Review by josh-mccrowell-braze — APPROVED** [review]

**Comment by josh-mccrowell-braze** [comment]
@cla-bot check

**Comment by josh-mccrowell-braze** [comment]
@cla-bot check

**Comment by josh-mccrowell-braze** [comment]
@cla-bot check

**Comment by josh-mccrowell-braze** [comment]
@cla-bot check

**Comment by josh-mccrowell-braze** [comment]
@cla-bot check

**Comment by josh-mccrowell-braze** [comment]
I hate you CLA bot, I hate you

**Comment by josh-mccrowell-braze** [comment]
@cla-bot check

**Comment by josh-mccrowell-braze** [comment]
@cla-bot check

**Comment by lydia-xie** [comment]
@cla-bot check

**Comment by lydia-xie** [comment]
@cla-bot check

**Comment by lydia-xie** [comment]
@cla-bot check

**Comment by lydia-xie** [comment]
@cla-bot check

**Comment by lydia-xie** [comment]
@cla-bot check

**Comment by lydia-xie** [comment]
> The cla-bot has been summoned, and re-checked this pull request!

@cla-bot check

**Comment by lydia-xie** [comment]
**_@cla-bot check_**

**Comment by iaraltkorn** [comment]
@braze-inc/docs-team @josh-mccrowell-braze when can we expect the PR to be merged? thanks

**Comment by josh-mccrowell-braze** [comment]
@iaraltkorn The CLA bot is preventing this from being merged at present. Technical difficulties. Please stand by.

**Comment by lydia-xie** [comment]
@cla-bot check

**Comment by lydia-xie** [comment]
@cla-bot check

**Comment by lydia-xie** [comment]
@cla-bot check

**Comment by lydia-xie** [comment]
@cla-bot check

**Comment by lydia-xie** [comment]
@cla-bot check

**Comment by lydia-xie** [comment]
@cla-bot check

**Comment by lydia-xie** [comment]
Closing this PR as these changes have been merged in this [new PR](https://github.com/braze-inc/braze-docs/pull/8028) due to the cla-bot issue

**Pull request closed without merging**